tools/tscdocgen/templates/nodejs.tmpl

```diff
@@ -16,17 +16,22 @@ git_sha: "{{GitSha}}"
 {{/PackageComment}}
 {{#Package}}
 {{=<% %>=}}
-{{< langchoose nodeonly >}}
+{{< chooser language "javascript,typescript" >}}
```

Comment:
Actually, we will indeed show _nothing_ in this case, because they're currently written to honor whatever's set globally (and to apply the selection globally on click). I totally didn't think of that use case. 🤔 

I think if we had the first occurring "tab" be active by default, and the first occurring choosable be active as well, then that'd work, but then it'd still be a bit odd to have selected Python somewhere, navigated to a page with only TypeScript and JavaScript, clicked JavaScript, and then you're in "JavaScript mode" from that point forward. Maybe you could designate a chooser as not globally applicable? This seems a little tricky.

Reply:
This has been addressed -- the first tab and choosable option will now be shown when the globally selected one isn't available in the list.